src/binaries/query-enterprise/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "databend-query-enterprise"
+description = "databend command line tools"
+version = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+publish = { workspace = true }
+edition = { workspace = true }
+
+[features]
+default = ["simd", "jemalloc", "databend-enterprise"]
```

Comment:
I don't think we should add an extra cargo file.

Why not use features to control that ?